bolt/test/AArch64/long-jmp-offset-boundary.s

```diff
@@ -0,0 +1,30 @@
+# This test checks long call negative offset boundary(0x8000000) for aarch64.
+
+# REQUIRES: system-linux
+
+# RUN: %clang %cflags %s -o %t.exe -nostartfiles -fuse-ld=lld -Wl,-z,now,-q \
```

Comment:
please use llvm-mc like in other tests to compile the object file. Also please rename main to _start. Also is -Wl,-z option really needed? -Wl,-q would be enough I think

Reply:
ok,thanks.